### PR TITLE
Add SSE communication facilities template to rx_block_cli

### PR DESCRIPTION
**Pull request by kostadin-damyanov-prime**

### CODE REVIEW

**Review by StanevPrime — CHANGES_REQUESTED** [review]
Please document the changes in the rx_bloc_cli

**Comment by kostadin-damyanov-prime** [comment]
> Please document the changes in the rx_bloc_cli

Updated CHANGELOG.md

File: packages/rx_bloc_cli/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
+## [x.x.x]
+* Added SSE communication facilities template
```
Comment (StanevPrime) [inline packages/rx_bloc_cli/CHANGELOG.md:2]: Let's ship the see and the patrol integration in one release

**Review by StanevPrime — CHANGES_REQUESTED** [review]
Could you please add a section using the moustache condition In the read me file? I guess this will be a good starting point from tech documentation perspective for the apps that need real time updates.

**Pull request merged**